doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -269,4 +296,50 @@ You are now all set to start the User OS (UOS)
 
 **Congratulations**, you are now watching the User OS booting up!
 
-.. _Clear Linux: https://clearlinux.org
+
+
+Enabling network sharing 
+************************
+
+after booting up SOS and UOS, you may need to run some basic test and install addational pakcages
```

Comment:
How do you know if you "may need to run some basic test and install additional packages"?  I thought this script wasn't needed anymore?

Reply:
no, this is needed for Ubuntu as the SOS.  f